src/MainWindow.ui

```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>946</width>
-    <height>632</height>
```

Comment:
Can you revert this please? It is an automatically updated value, when you open the UI file it is updating it based on your Qt window size.